bindings/cpp/CMakeLists.txt

```
@@ -0,0 +1,202 @@
+# Copyright 2025 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required(VERSION 3.21)
+project(svs_runtime VERSION 0.0.10 LANGUAGES CXX)
```

```suggestion
project(svs_runtime VERSION 0.0.11 LANGUAGES CXX)
```
Bump to 0.0.11? How is this used?

---

Cool, I'll look into adding this for our shared library as well